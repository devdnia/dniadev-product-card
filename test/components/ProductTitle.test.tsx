import React from 'react';
import renderer from 'react-test-renderer';
import { ProductTitle } from '../../src/components/ProductTitle';


describe('ProducTitle', () =>{

    test('Debe de mostrar el compomente correctamente con el título personalizado', () =>{

        const wrapper = renderer.create(
            <ProductTitle  title="Título personalizado" />
        )
        console.log(wrapper.toJSON() );
    })
})
