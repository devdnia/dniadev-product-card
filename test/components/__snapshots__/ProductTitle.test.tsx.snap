// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProducTitle Debe de mostrar el compomente correctamente con el título personalizado 1`] = `
<span
  className="productDescription custom-class"
>
  Título personalizado
</span>
`;

exports[`ProducTitle Debe de mostrar el componente con el nombre del producto 1`] = `
<div
  className="productCard undefined"
>
  <span
    className="productDescription undefined"
  >
    Coffee
  </span>
</div>
`;
